src/hatti/views/table.cljs

```diff
@@ -119,17 +123,28 @@
   ([form] (flat-form->sg-columns form true))
   ([form get-label?] (flat-form->sg-columns form get-label? nil))
   ([form get-label? language & {:keys [is-filtered-dataview? owner]}]
-   (let [columns (for [field (all-fields form :is-filtered-dataview?
+   (let [has-hxl? (any? false? (map #(nil? (-> % :instance :hxl)) form))
+         columns (for [field (all-fields form :is-filtered-dataview?
                                          is-filtered-dataview?)]
-                   (let [{:keys [name type full-name]} field
-                         label (if get-label? (get-label field language) name)]
-                     {:id name :field full-name :type type
-                      :name label :toolTip label :sortable true
+                   (let [{:keys [name type full-name]
+                          {:keys [hxl]} :instance} field
+                         label (if get-label? (get-label field language) name)
+                         column-class (get-column-class field)]
+                     {:id name
+                      :field full-name
+                      :type type
+                      :name
+                      (str "<div class=\"" column-class "\">" label "</div>"
```

Comment:
this could be made neater be pulling this out into its own function and calling the function here instead to return `name`

e.g.
```clojure
(defn get-column-name [label hxl]
    (str "<div class=\"" column-class "\">" label "</div>"
          (when hxl
              (str "<div class=\"hxl-row\">" hxl "</div>")))
```